workflow-templates/common-auto-update.yml

```diff
@@ -73,33 +85,6 @@ jobs:
       - name: Determine file extensions for github action files
         run: echo FILE_TYPE=$(ls ${{ env.TARGET_DIR }}/auto-update* |  tr . \\n | tail -n1) >> ${GITHUB_ENV}
 
-      - name: Set project type to gradle
-        run: echo PROJECT_TYPE="unknown" >> ${GITHUB_ENV}
-
-      - name: Test project-type (for gradle)
-        id: determine-type-gradle
-        run: echo cnt_validations=$(ls ${{ env.TARGET_DIR }}/deploy* | xargs grep -w 'gradle/wrapper-validation-action' | wc -l) >> ${GITHUB_OUTPUT}
-
-      - name: Set project type to gradle
-        if: steps.determine-type-gradle.outputs.cnt_validations != '0'
-        run: echo PROJECT_TYPE=gradle >> ${GITHUB_ENV}
-
-      - name: Test project-type (for yarn classic)
-        id: determine-type-yarn
-        run: echo cnt_validations=$(ls ${{ env.TARGET_DIR }}/deploy* | xargs grep -w 'NPM_REGISTRY_AUTH' | wc -l) >> ${GITHUB_OUTPUT}
-
-      - name: Set project type to yarn classic
-        if: steps.determine-type-yarn.outputs.cnt_validations != '0'
-        run: echo PROJECT_TYPE=yarn >> ${GITHUB_ENV}
-
-      - name: Test project-type (for yarn2+)
-        id: determine-type-yarn2
-        run: echo cnt_validations=$(ls ${{ env.TARGET_DIR }}/deploy* | xargs grep -w 'YARN_NPM_AUTH_TOKEN' | wc -l) >> ${GITHUB_OUTPUT}
-
-      - name: Set project type to yarn2+
-        if: steps.determine-type-yarn2.outputs.cnt_validations != '0'
-        run: echo PROJECT_TYPE=yarn2 >> ${GITHUB_ENV}
-
```

Comment:
IMHO: auto-update.yml installs workflows, like deploy.yml. But it has to look in deploy.yml to see version? But we are installing the deploy.yml workflow. For initial installation that does not work. Using PROJECT_TYPE in workflow-config.env seems way easier.

Reply:
That was the reason why I did not allow first installation using the auto-update mechanism.
In my opinion we should keep the workflow config down to a minimum (although sometimes we are missing things like the node version for JS/yarn/yarn2 projects).

Reply:
@NikoKrismerInnio What are your thoughts on using the auto-update for first installation? I think the initialization of repositories gets way more easier, since you only have to add 1 action (auto-update) and the `workflow-config.env` file.